start-testnet-reth-miner-node.sh

```
@@ -0,0 +1,51 @@
+# !/bin/sh
+ETH_DATA_DIR=.scalerized/eth
+JWT_PATH=testing/files/jwt.hex
+ETH_GENESIS_PATH=testing/files/eth-genesis.json
+retries=3
+delay=3
+enode_url=""
+
+get_enode() {
+  for ((i = 1; i <= retries; i++)); do
```

Shellcheck (0.10.0)

[warning] In POSIX sh, arithmetic for loops are undefined.

(SC3005)